docs/content/guide/external.ngdoc

```diff
@@ -0,0 +1,146 @@
+@ngdoc overview
+@name External Resources
+@sortOrder 560
+@description
+
+This is a collection of external, 3rd party resources for learning and developing Angular.
+
+## Articles, Videos, and Projects
+
+### Introductory Material
+
+* [10 Reasons Why You Should Use AngularJS](http://www.sitepoint.com/10-reasons-use-angularjs/)
+* [10 Reasons Why Developers Should Learn AngularJS](http://wintellect.com/blogs/jlikness/10-reasons-web-developers-should-learn-angularjs)
+* [Design Principles of AngularJS (video)](https://www.youtube.com/watch?v=HCR7i5F5L8c)
+* [Fundamentals in 60 Minutes (video)](http://www.youtube.com/watch?v=i9MHigUZKEM)
+* [For folks with a jQuery background](http://stackoverflow.com/questions/14994391/how-do-i-think-in-angularjs-if-i-have-a-jquery-background)
+
+### Specific Topics
+
+#### Application Structure & Style Guides
+
+* [Architecture, file structure, components, one-way dataflow and best practices](https://github.com/toddmotto/angular-styleguide)
+* [Angular Styleguide](https://github.com/johnpapa/angular-styleguide)
```

Comment:
This now contains both ng1 and ng2 guides. The ng1 guide can be found at https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md. 

In any case, I would move this link to the top, since this is the de-facto styleguide for ng1.